quota/v1/0000_03_quota-openshift_01_clusterresourcequota.crd.yaml-merge-patch

```diff
@@ -8,7 +8,6 @@ spec:
               properties:
                 hard:
                   additionalProperties:
-                    anyOf:
-                    - type: string
-                    - type: integer
-                    type: null
+                    type: ""
```

Comment:
I had null before to remove the type. But empty string should work as well I guess.

Reply:
For posterity here's why: beforehand the merge was done with crd-schema-gen, which used `gopkg.in/yaml.v2`, now we're using `yq` and that is using `gopkg.in/imdario/mergo.v0` and for some reason it's treating `null` as if not specified so it's not doing anything, I've tried in a live cluster, setting empty string works the same way, because empty and unset for `omitempty` fields is the same.